librz/include/rz_cons.h

```diff
@@ -846,6 +846,20 @@ typedef struct rz_bar_options_t {
 	bool color; //<< Use colors
 } RzBarOptions;
 
+typedef struct rz_i_histogram_t {
```

Comment:
Please rename this one too - `RzHistogramInteractive`

Reply:
done